packages/logger/src/logger.ts

```diff
@@ -56,42 +86,40 @@ export type LogHandler = (
   ...args: unknown[]
 ) => void;
 
+/**
+ * By default, `console.debug` is not displayed in the developer console (in
+ * chrome). To avoid forcing users to have to opt-in to these logs twice
+ * (i.e. once for firebase, and once in the console), we are sending `DEBUG`
+ * logs to the `console.log` function.
+ */
+const ConsoleMethod = {
+  [LogLevel.DEBUG]: 'log',
+  [LogLevel.VERBOSE]: 'log',
+  [LogLevel.INFO]: 'info',
+  [LogLevel.WARN]: 'warn',
+  [LogLevel.ERROR]: 'error'
+};
+
 /**
  * The default log handler will forward DEBUG, VERBOSE, INFO, WARN, and ERROR
  * messages on to their corresponding console counterparts (if the log method
  * is supported by the current log level)
  */
-const defaultLogHandler: LogHandler = (instance, logType, ...args): void => {
+const defaultLogHandler: LogHandler = (instance, logType, message): void => {
```

Comment:
Aren't we going to miss the additional args in log messages?

Reply:
I think this must have been overlooked, fixed.